opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -182,15 +180,12 @@ class CpGridData
     friend
     void ::check_global_refine(const Dune::CpGrid&,
                                const Dune::CpGrid&);
-
-    friend
-    void ::lookup_check(const Dune::CpGrid&);
     friend
     void ::fieldProp_check(const Dune::CpGrid& grid, Opm::EclipseGrid eclGrid, std::string deck_string);
     friend
     void ::testInactiveCellsLgrs(const std::string&,
-                           const std::vector<std::array<int,3>>&,
-                           const std::vector<std::array<int,3>>&,
+                                 const std::vector<std::array<int,3>>&,
+                                 const std::vector<std::array<int,3>>&,
```

Comment:
Is indentation correct, below here?

Reply:
Ops, I must have one that without intention. I'll correct indentation